tasks/borg-client.yml

```diff
@@ -35,15 +35,13 @@
         {% endif %}
   with_items: "{{ borgbackup_servers }}"
 
+- debug: msg="{{ sshkey.stdout }}"
+
 - name: client | put sshpubkey on the normal backupserver
   authorized_key:
     user: "{{ item.user }}"
     key: "{{ sshkey.stdout }}"
-    key_options: >
```

Comment:
@dverhelst I'd vote to split this up to make it easier to read, just need to remove the single quotes at the beginning and the end, otherwise it literally reads it as a multiline string.

Reply:
@noplanman , I really appreciate your comments on all of this. Can you pick this one up in a next PR? 
I'm having trouble enough as is getting this one through. My idea was to have decent testing before adding all of these wonderful improvements you are proposing.

Reply:
@dverhelst Of course!
I have a bunch of changes / fixes lined up already for a new PR, waiting for this one to get in first though.

Great work on the testing front. I need to check out how it works and learn the molecule approach, looks interesting.